lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.html

```diff
@@ -10,12 +10,12 @@
           [attr.title]="widget.getDisplayValue() | async"
           #menuTrigger="matMenuTrigger">
     <span class="adf-search-filter-placeholder">
-        <span class="adf-search-filter-ellipsis">{{ category.name | translate }}</span>
-        <ng-container *ngIf="widget.getDisplayValue() | async">:</ng-container>
+        <span class="adf-search-filter-ellipsis">{{ category.name | translate }}:</span>
     </span>
-    <span class="adf-search-filter-ellipsis" *ngIf="widget.getDisplayValue() | async as displayValue">
+    <span class="adf-search-filter-ellipsis adf-filter-value" *ngIf="widget.getDisplayValue() | async as displayValue; else showAny">
         &nbsp;{{ displayValue | translate }}
```

Comment:
Do we need the nbsp here? It can cause some troubles when trying to debug linting errors around these lines.

Reply:
We need nbsp here to prevent breaking 1 facet on separate lines, I am not sure about possible linting errors, linter is green now, and nbsp was already written in this component for a long time without problems

Reply:
Ahh, ok. Then this is fine. Marked as resolved